src/App.tsx

```diff
@@ -10,9 +13,16 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+
+            <Route path="/home" element={<Navigate to="/" replace />} />
+
+            <Route path="/people/" element={<PeoplePage />} />
```

Comment:
As a general rule of thumb implementation: 
```
              <Route path="people">
                <Route path=":slug?" element={<PeoplePage />} />
              </Route>
```

This way you implicitly state that 'hey, there is a root page and it may include slug.